integration-tests/sanity/sanity_test.go

```diff
@@ -168,7 +168,7 @@ var _ = Describe("Sanity", func() {
 
 		It("should delete cstor volume replica CR", func() {
 			// Getting the cstor volume replica artifacts
-			artifact, err := artifacts.GetArtifactUnstructured(artifacts.CVRArtifact)
+			artifact, err := artifacts.GetArtifactUnstructuredFromFile(artifacts.CVRArtifact)
```

Comment:
Mark a TODO as these things should not be in artifacts yaml.
Only the openebs operator contents will reside in artifacts yaml

Reply:
added